os/net/netmgr/netdev_stats.h

```diff
@@ -0,0 +1,38 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __NETMGR_STATS_H__
+#define __NETMGR_STATS_H__
+
+extern uint32_t g_link_recv_byte;
+extern uint32_t g_link_recv_cnt;
+extern uint32_t g_link_recv_err;
+
+extern uint32_t g_app_recv_byte;
+extern uint32_t g_app_recv_cnt;
```

Comment:
In my opinion, you can remove these values here,
And you had better use these values as a private variable.
I mean make some functions instead of common MACRO

Reply:
I applied lwip stats style. please check lwip/stats.h. I'll modify it later if it has problems.